Python-Home-Challenges/recurring word.py

```diff
@@ -0,0 +1,22 @@
+
+where_file = input('Enter the file route: ')
+my_file = open(where_file,"r")
+print("Your file text is:")
+for x in my_file:
+  print(x)
+print("---------------")
```

Comment:
this output is not required

Reply:
that was just for my debugging, will be deleted.